Add SSH documentation to the website. (#692)
ServiceWeaver/weaver: website/docs.md

[Kubernetes][kubernetes], [Docker][docker], etc.

## Getting Started

Prerequisites:
* A set of machines reachable via `ssh`.
* You may want to set up passwordless `ssh` between your machines, otherwise you
will have to type the password for each machine when you deploy/stop an application.

Consider again the "Hello, World!" Service Weaver application from the [Step by
Step Tutorial](#step-by-step-tutorial) section. The application runs an HTTP
server on a listener named `hello` with a `/hello?name=<name>` endpoint that
returns a `Hello, <name>!` greeting. To deploy this application using the `SSH`
deployer, first create a [Service Weaver application config file](#config-files),
say `weaver.toml`, with the following contents:

```toml
[serviceweaver]
binary = "./hello"

[ssh]
listeners.hello = {address = "localhost:9000"}
locations = "./ssh_locations.txt"
```

The `[serviceweaver]` section of the config file specifies the compiled Service
Weaver binary. The `[ssh]` section contains the set of machines where your
application should be deployed, and per listener configuration. The set of
machines are specified as follows:

```txt

> // ssh_locations.txt

Author: Yeah, I don't think so. Reworded.

10.100.12.31
10.100.12.32
10.100.12.33
...
```

Deploy the application using `weaver ssh deploy`:

```console
$ weaver ssh deploy weaver.toml
```

When `weaver ssh deploy` terminates (e.g., when you press `ctrl+c`), the
application is destroyed and all processes are terminated.

## Logging

> `weaver ssh logs` logs to stdout. Refer to `weaver ssh logs --help` for a full
> explanation of the query language, along with many more examples.

Reviewer: ```suggestion
`weaver ssh logs` logs to stdout. Refer to `weaver ssh logs --help` for details.
```

Author: Done
